.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,14 @@
+blank_issues_enabled: false
```

Comment:
Do I understand this option correctly that it prevents a user from filing issues that are not bug reports?

Reply:
I meant the specific line I commented on 😅  but thanks for the link, that answers my question:

> You can encourage contributors to use issue templates by setting blank_issues_enabled to false. If you set blank_issues_enabled to true, people will have the option to open a blank issue.